code/modules/ruins/spaceruin_code/clownstation.dm

```diff
@@ -0,0 +1,56 @@
+
+/obj/item/paper/fluff/ruins/clownstation/honkprayer
+	name = "prayer"
+	info = "I honk myself with the sacred honk. It is the same honk that honked the universe into existence. It is the honk that honked the breath of life into the first clown. It is the eternal honk. Amen."
+
+/obj/item/paper/fluff/ruins/clownstation/bananaprayer
+	name = "prayer"
+	info = "Hail the banana tree, full of pranks. The Honkmother is with thee. Blessed art thou amongst plants, and blessed is the fruit of thy branch, the banana. Holy banana tree, Mother of pranks, grow for us prankers, now and at the hour of our death, Amen."
+
+/obj/item/paper/fluff/ruins/clownstation/gospel
+	name = "reading from the Honkspel of Jhonk"
+	info = "The Honkmother did take her Horn, and she did Honk it, and it was a magnificent Honk. And the Honk Honked deep into the hearts and minds of the Honkpostles. This is the word of the Honkmother."
+
+/obj/item/paper/fluff/ruins/clownstation/honkilious
+	name = "reading from the Honkspel of Honksodus"
+	info = "In the early years of the universe all was good. Clowns made pilgrimage to the Honkly land and lived together on the clown planet, free from harm and aple to prank honkfully. Honkilious, the clown king, ruled a golden empire. However, honkilious was greedy. He taxed his people, taking a large portion of their bananas. The honkmother was not pleased. She stole the color from Honkilious's body, and ripped the voice from his throat. Banished into mimery, Honkilious grew furious. He demanded that the people give him all of their bananas, so that he could indulge in at least one honkfull pleasure. As Honkilious bit into his first banana, its color too drained away. Honkilious watched in horror as his bananas became sickly and pale. The mimana was born. Honkilious honked his horn, but no sound came out. Depressed beyond belief, Honkilious threw himself into space, wishing for death. The Honkmother saw through Honkilious's plan and cursed him with eternal life. For eternal life as a mime is eternal suffering for a clown. Disgusted by the greed of her creation, the Honkmother forbid any one clown from having too much power. Government buildings crumbled, and leaders went up in yellow flames. However, the abolition of government did not deter the greedy clowns. They quarreled daily over bananas, and were never satisfied with any sum. The Honkmother took all but the most virtuous clowns and stripped them from the clown planet, exiling them to various planets across the cosmos. Thus, it is true that we have not newly become clowns, but rather, we have rediscovered our clownish ways so that we may return to what was once ours."

```

Comment:
capitalize proper names reeeeeeee

I stopped reading partway through this line because I wanna save some L O R E for when I find the ruin in-game. I'd recommend proofreading these notes again before adding them to the game. Good luck with this clown lore project; I like it.